### PR TITLE
#16990: 'View on Etherscan' now works for other networks

### PR DESCRIPTION
**Pull request by rahulsnkr**
Resolves brave/brave-browser#16990. 

https://user-images.githubusercontent.com/28645536/126897594-3b6b5eda-56bd-49e9-82d6-d9756759fc4d.mp4

cc @bbondy

### CODE REVIEW

**Review by onyb — CHANGES_REQUESTED** [review]
Looks great! It actually makes the `getEtherScanNetworkIdentifier` function useless. Can you please also remove it and fix the corresponding usage?

https://github.com/brave/ethereum-remote-client/blob/08776ca72954716d6492b6ad7aab6f5bdfd9f249/ui/lib/etherscan-prefix-for-network.js#L24-L43

Then, replace `networkId` with `network` here:

https://github.com/brave/ethereum-remote-client/blob/08776ca72954716d6492b6ad7aab6f5bdfd9f249/ui/app/pages/swap/wait-for-transaction/component.js#L56

Please do a separate commit for the suggested changes.

**Comment by rahulsnkr** [comment]
@onyb thanks for the review! :)
Updated.

**Review by onyb — APPROVED** [review]

File: ui/lib/etherscan-prefix-for-network.js [hunk 1]
```diff
@@ -8,37 +8,19 @@ import * as networkEnums from '../../app/scripts/controllers/network/enums'
  */
 export function getEtherscanNetworkPrefix (networkId) {
   switch (networkId) {
+    case networkEnums.ROPSTEN_CHAIN_ID:
```
Comment (onyb) [inline ui/lib/etherscan-prefix-for-network.js:11]: I think we also need a case for `networkEnums.ROPSTEN` to replace the functionality of `getEtherScanNetworkIdentifier`.

**Review by onyb — CHANGES_REQUESTED** [review]
One last remark to address, and then it's good to merge.

File: ui/lib/etherscan-prefix-for-network.js (hunk 1 above)
Comment (rahulsnkr) [reply ui/lib/etherscan-prefix-for-network.js:11]: do we need this case just for `ROPSTEN`, or for the other networks as well?

Comment (onyb) [reply ui/lib/etherscan-prefix-for-network.js:11]: Sorry, I didn't specify that. It must be for all networks.

Comment (rahulsnkr) [reply ui/lib/etherscan-prefix-for-network.js:12]: Updated.

**Review by onyb — APPROVED** [review]

**Pull request merged**